lib/flipper/api/v1/actions/groups_gate.rb

```diff
@@ -0,0 +1,41 @@
+require 'flipper/api/action'
+
+module Flipper
+  module Api
+    module V1
+      module Actions
+        class GroupsGate < Api::Action
+          route %r{api/v1/features/[^/]*/groups/[^/]*/(enable|disable)/?\Z}
+
+          def put
+            if feature_names.include?(feature_name) && Flipper.group_names.include?(group_name)
```

Comment:
Perhaps we could use `Flipper.group_exists?` instead of checking the collecting in `group_names`?